lesson01/assignment1/ES6Methods.js

```diff
@@ -0,0 +1,86 @@
+
+//Setup JQuery function
+$(function () {
```

Comment:
`(function () { }` is the function that executes when the DOM for the page completes loading. All the processing code for the page can be inside it- no need to create separate ones for each form.